Library/Homebrew/env_config.rb

```diff
@@ -260,6 +260,11 @@ module EnvConfig
         description: "If set, use Pry for the `brew irb` command.",
         boolean:     true,
       },
+      HOMEBREW_SIMULATE_MACOS_ON_LINUX:       {
```

Comment:
Is there any situation where you'd want just `HOMEBREW_SIMULATE_MACOS_ON_LINUX` and not `HOMEBREW_FORCE_HOMEBREW_ON_LINUX`? If not, could we make `HOMEBREW_SIMULATE_MACOS_ON_LINUX` automatically set/imply `HOMEBREW_FORCE_HOMEBREW_ON_LINUX`?

Reply:
> Is there any situation where you'd want just `HOMEBREW_SIMULATE_MACOS_ON_LINUX` and not `HOMEBREW_FORCE_HOMEBREW_ON_LINUX`?

Probably not - or at least not for any of our use cases. In theory, `HOMEBREW_SIMULATE_MACOS_ON_LINUX` should work on `linuxbrew-core` and pass `brew audit` without issue - but that's not something we particularly care about.

> make `HOMEBREW_SIMULATE_MACOS_ON_LINUX` automatically set/imply `HOMEBREW_FORCE_HOMEBREW_ON_LINUX`?

Yeah that makes sense.

Reply:
> Yeah that makes sense.

👍🏻 cool, let's do/document that then.